### DISCUSSION TITLE
How to save a project and how to restore it ?

### DISCUSSION THREAD

**Discussion by PhilDu77** [nocodb/nocodb#7842] [General]
Considering a project is set of form or grid views and all the settings like BD connections.
Is there somewhere a fonction to save all the project into the file for exemple ? and also the same to restore a project from a file.
thanks.  
the backup(or restore) of the data from a MariaDB database is done  independently

**Comment by dstala** [comment]
@PhilDu77 You can create a backup of your root DB (NC_DB if you had configured, noco.db sqlite file otherwise). This will copy all the bases though, not just one particular base. Export feature on the UI per base is in our roadmap- not available in the UI yet.
